fuse_constraints/include/fuse_constraints/relative_constraint_impl.h

```diff
@@ -48,11 +48,12 @@ namespace fuse_constraints
 
 template<class Variable>
 RelativeConstraint<Variable>::RelativeConstraint(
+  const std::string& source,
   const Variable& variable1,
   const Variable& variable2,
   const fuse_core::VectorXd& delta,
   const fuse_core::MatrixXd& covariance) :
-    fuse_core::Constraint{variable1.uuid(), variable2.uuid()},
+    fuse_core::Constraint(source, {variable1.uuid(), variable2.uuid()}),  // NOLINT(whitespace/braces)
```

Comment:
As mentioned before, I think this should work: `fuse_core::Constraint{source, variable1.uuid(), variable2.uuid()},` (same below)